services/auth.js

```diff
@@ -0,0 +1,61 @@
+import { Strategy as JwtStrategy, ExtractJwt } from "passport-jwt";
+import "dotenv/config";
+import passport from "passport";
+import bcrypt from "bcrypt";
+import db from "../database.js";
+import jwt from "jsonwebtoken";
+
+// Jwt strategy
+const opts = {
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+  secretOrKey: process.env.JWT_SECRET,
+};
+
+const jwtStrategy = new JwtStrategy(opts, async (payload, done) => {
+  const user = db.getUserByEmail(payload.email);
```

Comment:
I think db.getUserByEmail(payload.email) should be async func (maybe not now but in future)